### PR TITLE
Change most reference to slack to point to the forum

### PR DESCRIPTION
**Pull request by opatut**
Also updates the community pages, the slack entrypage, the landing page, adds `/contact` and `/kontakt` aliases to the community page, and changes lots of wording for referencing to the community and ways to contact it.

### CODE REVIEW

File: content/docs/user-guide/simra/index.de.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Prinzipiell gibt es 3 Features, von denen wir nur 2 unterstützen können:
 
 Wir haben uns dazu entschieden, Bluetooth *vorläufig* auszubauen, da es das am
```
Comment (gluap) [inline content/docs/user-guide/simra/index.de.md:18]: DIeser Block ist im anderen Review schon entfernt.

File: content/_index.de.html [hunk 2]
```diff
@@ -101,7 +115,7 @@ <h2>Visualisierung</h2>
 
 {{< yt id=izUD16ffs_Q thumbnail="/thumbnails/izUD16ffs_Q.jpg" >}}
 
-Wer noch Ideen zur Visualisierung, Automatisierung, Programmierung, Auswertung, etc. hat oder einfach am Projekt interessiert ist und mitgestalten will, ist herzlich in der Community via <a href="{{< relref "/slack" >}}">Slack</a> eingeladen.
+Wer noch Ideen zur Visualisierung, Automatisierung, Programmierung, Auswertung, etc. hat oder einfach am Projekt interessiert ist und mitgestalten will, ist herzlich <a href="{{< relref "/community" >}}">in der Community</a> eingeladen.
```
Comment (gluap) [inline content/_index.de.html:118]: ... in **die** community eingeladen ...

**Review by gluap — CHANGES_REQUESTED** [review]
I apologize for my edit requiring two changes and the messy review style mixing review and comments, This is my next-to-first github review as reviewer. 

One thing is left: There's an additional mention of the slack group order channel in 
``content/faq/build/where-do-i-get-a-sensor.de.md``

I proof-read the text and built the change with hugo successfully.

File: content/docs/user-guide/simra/index.de.md (hunk 1 above)
Comment (opatut) [reply content/docs/user-guide/simra/index.de.md:18]: Ja, darum lasse ich ihn hier drin. Git wird das schon mergen.

File: content/_index.en.html [hunk 3]
```diff
@@ -8,19 +8,51 @@
...
+{{% /blocks/feature %}}
+
+{{% blocks/feature icon="fas fa-biking" title="Build & measure" url="/bauanleitung" url_text="Zur Bauanleitung" %}}
+Build your own sensoric device. We will support you as much as we can. How
+about joining a local group of builders and bikers, and join a collective order
+to save on parts?
+{{% /blocks/feature %}}
+
+{{% blocks/feature icon="fab fa-github" title="Develop" url="https://github.com/openbikesensor" url_text="Zu den GitHub-Repositories" %}}
+Whatever we create, we make open source. Everything is available on GitHub,
```
[the remark this answers is not in the document]
Comment (opatut) [reply content/_index.en.html:48]: > We publish everything we create as open source.

I went for this, a bit shorter. Good suggestion though ;)

**Comment by opatut** [comment]
Thanks @gluap, I applied your suggestions, or at least most of them ;) Please recheck?

**Review by gluap — APPROVED** [review]

**Pull request merged**